src/WebSockets/WebSocketHandler.php

```diff
@@ -26,6 +27,20 @@ public function __construct(ChannelManager $channelManager)
         $this->channelManager = $channelManager;
     }
 
+    /*
+     * Match pusher channel protocols
+     * @link https://pusher.com/docs/channels/library_auth_reference/pusher-websockets-protocol
+     */
+    public function getSubProtocols()
```

Comment:
Note that I've set the supported protocols to `4...7` as the Pusher docs say.

From what I could find in the swift SDK codebase, protocol version 7 has been in there from the start (initial commit), which was 6 years ago.